internal/commands/run.go

```diff
@@ -33,28 +33,22 @@ const (
 	nameKindReference
 )
 
-const longDescription = `Install an application.
-By default, the application definition in the current directory will be
-installed. The APP_NAME can also be:
-- a path to a Docker Application definition (.dockerapp) or a CNAB bundle.json
-- a registry Application Package reference`
+const longDescription = `Run an application based on a docker app image.`
 
-const example = `$ docker app install myapp.dockerapp --name myinstallation --target-context=mycontext
-$ docker app install myrepo/myapp:mytag --name myinstallation --target-context=mycontext
-$ docker app install bundle.json --name myinstallation --credential-set=mycredentials.yml`
+const example = `$ docker app run myrepo/myapp:mytag --name myinstallation --target-context=mycontext`
 
-func installCmd(dockerCli command.Cli) *cobra.Command {
-	var opts installOptions
+func runCmd(dockerCli command.Cli) *cobra.Command {
+	var opts runOptions
 
 	cmd := &cobra.Command{
-		Use:     "install [APP_NAME] [--name INSTALLATION_NAME] [--target-context TARGET_CONTEXT] [OPTIONS]",
+		Use:     "run [APP_NAME] [--name INSTALLATION_NAME] [--target-context TARGET_CONTEXT] [OPTIONS]",
 		Aliases: []string{"deploy"},
-		Short:   "Install an application",
+		Short:   "Run an application",
 		Long:    longDescription,
 		Example: example,
 		Args:    cli.RequiresMaxArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runInstall(dockerCli, firstOrEmpty(args), opts)
+			return runRun(dockerCli, firstOrEmpty(args), opts)
```

Comment:
runRun! 🏃

Reply:
Shouldn't be `firstOrEmpty(args)` I think right? Now the command needs the app image reference?

Reply:
Mayeb I should rename `RunForestRun` ?

Reply:
or `RunLolaRun`?